test/src/actions/android/welcome.actions.js

```diff
@@ -34,8 +34,9 @@ class WelcomeActions {
         await ActionHelper.click(Pages.welcomePage.secureNow);
         await ActionHelper.click(Pages.welcomePage.rightNavigationArrow);
         await ActionHelper.click(Pages.welcomePage.rightNavigationArrow);
+        await ActionHelper.pause(1000)
```

Comment:
I think it's better to set an optional parameter in the click() in order to pause the driver a certain amount of time on demand
`    static async click(locator, waitTimeInMiliSeconds = 100) {
        try {
            const elem = await $(locator);
            await this.waitForDisplayed(elem);
            await elem.click();
            await this.pause(waitTimeInMiliSeconds);
        } catch (error) {
            console.log("Error while clicking on element: " + error);
            throw new Error("Error while clicking on element: " + error);
        }
    }`
what do you think?

Reply:
function added to wdio-common